glue_vispy_viewers/common/vispy_widget.py

```diff
@@ -32,7 +33,7 @@ def __init__(self, parent=None):
         self.emulate_texture = (sys.platform == 'win32' and
                                 sys.version_info[0] < 3)
 
-        self.scene_transform = scene.STTransform()
+        self.axis_transform = scene.STTransform()
```

Comment:
@astrofrog I found it's a little confusing for this variable name coz this `self.scene_transform` is only used for setting the axis's transform so I changed it's name to make it more clear, I can change it back if we want to keep it :)